### PR TITLE
CY-1487 - Export script for publishing version to NPM

### PR DESCRIPTION
**Pull request by qooban**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Scripts
 
+### create-version.sh
+
+Provides helper function for creating new version of NPM package by creating new branch with version change commit and pushing that branch to the remote repository.
```
Comment (tyacbovi) [inline scripts/README.md]: Creating a new

File: scripts/create-version.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
```
Comment (tyacbovi) [inline scripts/create-version.sh]: Add set -e

File: scripts/README.md (hunk 1 above)
Comment (qooban) [reply scripts/README.md]: Fixed.

File: scripts/create-version.sh (hunk 2 above)
Comment (qooban) [reply scripts/create-version.sh]: Added.

**Pull request merged**